crates/modelardb_compression/src/lib.rs

```diff
@@ -976,7 +979,10 @@ pub mod test_util {
             }
             // Generates linear data.
             StructureOfValues::Linear => {
-                let a: i64 = thread_rng().gen_range(-10..10);
+                let mut a: i64 = 0;
```

Comment:
Maybe add a comment with the reason why `a == 0` is not allowed.

Reply:
Fixed.